scripts/outgoing-mapping.R

```diff
@@ -26,17 +25,17 @@ make_geopackage(dat = phase1_priorities)
 
 
 ##add the tracks
-sf::read_sf("./data/habitat_confirmation_tracks.gpx", layer = "tracks") %>%
+sf::read_sf("./data/hab_con_tracks.gpx", layer = "tracks") %>%
```

Comment:
we should leave the name of our files equivalent from project to project and year to year as name changes can cause mega headaches